src/main/configureElectronApp.ts

```diff
@@ -84,29 +84,16 @@ const fatalError = (error: unknown) => {
     app.quit();
 };
 
-const initNrfutil = () => {
+const copyNrfutil = () => {
     const binName = `nrfutil${process.platform === 'win32' ? '.exe' : ''}`;
 
     const nrfutilBundled = path.join(getBundledResourcesDir(), binName);
     const nrfutilInAppPath = path.join(getUserDataDir(), binName);
```

Comment:
Do we see any value in this instead?
```suggestion
    const nrfutilInAppPath = path.join(getUserDataDir(), binName);
    const nrfutilBundledStats = fse.statSync(nrfutilBundled);

     const nrfutilInstalled = fse.existsSync(nrfutilInAppPath);
     const installedNrfutilNotSameAsTheBundledNrfutil =
         nrfutilInstalled &&
         Math.round(nrfutilBundledStats.mtimeMs) !==
             Math.round(fse.statSync(nrfutilInAppPath).mtimeMs);

     if (!nrfutilInstalled || installedNrfutilOlderThenBundledNrfutil) {
         fse.copyFileSync(nrfutilBundled, nrfutilInAppPath);
         fse.utimes(
             nrfutilInAppPath,
             nrfutilBundledStats.atime,
             nrfutilBundledStats.mtime


         );
         logger.info('update nrfutil exe');
     }
```

Reply:
I think copying over the `nrfutil` wrapper is so fast, it is better to just do that every time instead of having all that code for a small optimisation.